### PR TITLE
[internal-fix] Refactor the vSphere 4.14 docs

### PR DESCRIPTION
**Pull request by dfitzmau**
Compare refactor effort with [4.15 vSphere](https://docs.openshift.com/container-platform/4.15/installing/installing_vsphere/preparing-to-install-on-vsphere.html) doc structure. Ensure 4.15 specific content is not backported to 4.14. Also ensure [original 4.14](https://docs.openshift.com/container-platform/4.14/installing/installing_vsphere/preparing-to-install-on-vsphere.html) content is transferred to the refactor PR.

Version(s):
4.14

Issue:
[OSDOCS-10048](https://issues.redhat.com/browse/OSDOCS-10048)

Link to docs preview:
* [IPI vSphere](https://73685--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/ipi/ipi-vsphere-installation-reqs)
* [UPI vSphere](https://73685--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/upi/upi-vsphere-installation-reqs)

Additional information:
Test file: installing/installing_vsphere/preparing-to-install-on-vsphere.adoc

Screenshot from 2024-04-05 17-29-19

### CODE REVIEW

**Comment by dfitzmau** [comment]
Address the Minimum permissions for the storage components links in the following modules:

ipi-vsphere-installation-reqs.adoc

upi-vsphere-installation-reqs.adoc

**Comment by skopacz1** [comment]
Hey Darragh, I discussed this PR with the rest of the peer review squad, and we've decided that we won't review this PR as-is for a number of reasons:

1. It's an incredibly large PR that would require either coordination between several members of the squad or a large chunk of one reviewer's time, and in either case it's a huge burden for the team.
2. Even if we have the time, the size and complexity of this PR makes it prone to errors and missed things during review, especially since it moves files around, edits the topic map, and changes many xrefs.
3. We've discussed an issue like this before where @kalexand-rh said it would be reasonable to ask writers to break up very large PRs into smaller ones if possible.
4. Considering that Mike Pytlak performed the same kind of reorg for the [4.15+ vSphere docs](https://issues.redhat.com/browse/OSDOCS-8173), and he successfully did the work in many smaller chunks (see the subtasks in the linked JIRA issue, where I believe he dedicated a single PR for each subtask), it's clear that this work can be done in smaller chunks, so we do not feel that the burden of reviewing such a large PR is justified.

I don't think you need to perform this reorg in as many PRs as Mike did, but please break this work up into much smaller chunks. I think it would ensure better content and a better review process overall.

In the meantime I'll drop the peer-review-needed label, please add it back to the queue if you feel differently. Let me know if you have any questions, thanks!

File: networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-ingress-controller.adoc [hunk 1]
```diff
@@ -61,7 +61,7 @@ The Ingress Operator manages wildcard DNS. For more information, see the followi
 
 * xref:../../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metal[Installing a cluster on bare metal].
 
-* xref:../../installing/installing_vsphere/installing-vsphere.adoc#installing-vsphere[Installing a cluster on vSphere].
+* xref:../../installing/installing_vsphere/upi/installing-vsphere.adoc#installing-vsphere[Installing a cluster on vSphere].
```
Comment (JoeAldinger) [inline networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-ingress-controller.adoc:64]: I doubled checked this and it works, but I'm not sure why you wouldn't have to go three levels deep.

**Review by JoeAldinger — COMMENTED** [review]
I checked xrefs from where @bergerhoffer asked me to start to the bottom. All good on my end. I'd wait for her to finish.

**Review by bergerhoffer — COMMENTED** [review]
Everything looks great compared to the docs live on 4.15. Pointed out a few things, but those are all things that existed already in the 4.15+ docs, so I wouldn't make any changes in this PR. If anything, address those in a separate PR

File: networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-ingress-controller.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (bergerhoffer) [reply networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-ingress-controller.adoc:64]: This file is only 2 levels deep (networking/configuring_ingress_cluster_traffic), so this looks right to me.

The next file is 3 levels deep (observability/distr_tracing/distr_tracing_jaegar), so 3 backups is right.

**Comment by dfitzmau** [comment]
Thanks, @bergerhoffer and @JoeAldinger :pray: . I'll address the above feedback in a separate PR. 

/label merge-review-needed

**Pull request merged**